README.md

```diff
@@ -144,6 +66,12 @@ better flexibility in that regard. Go to your _Mailtrap account_ → _Email Send
 → _Sending Domains_ → _Your domain_ → _SMTP/API Settings_ to find the SMTP 
 configuration example.
 
+## Migration guide v1 → v2
+
+Change `Mailtrap::Sending::Client` to `Mailtrap::Client`.
+
+If you use classes which have `Sending` namespace, change them as well.
```

Comment:
Let's elaborate a bit more on how to change them

Reply:
a nicer approach is to add a compatibility client + a deprecation warning. but perhaps it's not needed because the library is small.

Reply:
It will be a new major version anyway. So it's "legitimate" to require such an upgrade 🙂

Reply:
oh! then yes for sure.